### ISSUE TITLE
1.11.10 calico cannot retrieve /certs/calico-client

### ISSUE THREAD

**Issue by pkutishch** [kubernetes/kops#7333]
**1. What `kops` version are you running? The command `kops version`, will display
 this information.**
Version 1.12.1 (git-e1c317f9c)

**2. What Kubernetes version are you running? `kubectl version` will print the
 version if a cluster is running or provide the Kubernetes version specified as
 a `kops` flag.**
1.10.11
**3. What cloud provider are you using?**
AWS
**4. What commands did you run?  What is the simplest way to reproduce this issue?**
kops upgrade cluster --yes
kops rolling-upgrade cluster --yes

**5. What happened after the commands executed?**
nodes failed to startup because of: 
```kubectl logs -f calico-node-mgjbk -n kube-system calico-node
2019-07-27 20:38:40.586 [INFO][9] startup.go 173: Early log level set to info
2019-07-27 20:38:40.586 [INFO][9] startup.go 184: NODENAME environment not specified - check HOSTNAME
2019-07-27 20:38:40.586 [INFO][9] client.go 202: Loading config from environment
Calico node failed to start
ERROR: Error accessing the Calico datastore: open /certs/calico-client.pem: no such file or directory```
**6. What did you expect to happen?**
Expect for nodes to comeup 


**9. Anything else do we need to know?**
Problem happened within upgrade from 1.10.11 -> 1.11.10 k8s version with adoption of etcd manager. 
Masters were rolled with "hammer" all masters at the same time, they came up normally, calico was working fine, after i started rolling nodes with the same version of kubernetes it appear that networking is broken for all workers as calico cannot retrieve certificate. 
to resolve it i decided to upgrade masters to 1.12.9 and right after problem was solved. 
Not sure why masters at 1.11 (Calico 2.6) aren't compatible with workers at the same version but with newer version of masters and calico it was fine.. 
We were already on etcdv3. 
Have someone run thru this issue?

**Comment by nwandj** [comment]
Hi, 
Have you found a fix to this?

Thanks,

**Comment by pkutishch** [comment]
The fix was to jump to 1.12.9 and 1.11.10 nodes started becoming ready and were able to connect to calico.. not sure why it happened.
